### PR TITLE
Maze

### PR DESCRIPTION
**Pull request by jdhj**
Please review this, @casey-collab

### CODE REVIEW

File: english/plugh/maze1/maze1.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+You are in a maze of twisty little passages, all different.
+
+Which way should we go?
+
+[north](maze1.md)
+[south](maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze1/maze1.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

File: english/plugh/maze1/maze1.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+[west](maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze1/maze1.md]: This link is also on the line directly after another link.

File: english/plugh/maze1/maze1.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+[east](maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze1/maze1.md]: This link is also on the line directly after another link.

File: english/plugh/maze1/maze1.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+[northwest](../maze2/maze2.md)
```
Comment (casey-collab) [inline english/plugh/maze1/maze1.md]: This link is also on the line directly after another link.

File: english/plugh/maze1/maze1.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+[northeast](maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze1/maze1.md]: This link is also on the line directly after another link.

File: english/plugh/maze1/maze1.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+[southwest](maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze1/maze1.md]: This link is also on the line directly after another link.

File: english/plugh/maze1/maze1.md [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+[southeast](maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze1/maze1.md]: This link is also on the line directly after another link.

File: english/plugh/maze1/maze1.md [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+[up](maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze1/maze1.md]: This link is also on the line directly after another link.

File: english/plugh/maze1/maze1.md [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+[down](maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze1/maze1.md]: This link is also on the line directly after another link.

File: english/plugh/maze2/maze2.md [hunk 10]
```diff
@@ -0,0 +1,14 @@
+You are in a maze of twisting little passages, all different.
+
+Which way should we go?
+
+[north](../maze1/maze1.md)
+[south](../maze1/maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze2/maze2.md]: This link is also on the line directly after another link.

File: english/plugh/maze2/maze2.md [hunk 11]
```diff
@@ -0,0 +1,14 @@
...
+[west](../maze1/maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze2/maze2.md]: This link is also on the line directly after another link.

File: english/plugh/maze2/maze2.md [hunk 12]
```diff
@@ -0,0 +1,14 @@
...
+[east](../maze1/maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze2/maze2.md]: This link is also on the line directly after another link.

File: english/plugh/maze2/maze2.md [hunk 13]
```diff
@@ -0,0 +1,14 @@
...
+[northwest](../maze1/maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze2/maze2.md]: This link is also on the line directly after another link.

File: english/plugh/maze2/maze2.md [hunk 14]
```diff
@@ -0,0 +1,14 @@
...
+[northeast](../maze1/maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze2/maze2.md]: This link is also on the line directly after another link.

File: english/plugh/maze2/maze2.md [hunk 15]
```diff
@@ -0,0 +1,14 @@
...
+[southwest](../maze1/maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze2/maze2.md]: This link is also on the line directly after another link.

File: english/plugh/maze2/maze2.md [hunk 16]
```diff
@@ -0,0 +1,14 @@
...
+[southeast](../maze3/maze3.md)
```
Comment (casey-collab) [inline english/plugh/maze2/maze2.md]: This link is also on the line directly after another link.

File: english/plugh/maze2/maze2.md [hunk 17]
```diff
@@ -0,0 +1,14 @@
...
+[up](../maze1/maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze2/maze2.md]: This link is also on the line directly after another link.

File: english/plugh/maze2/maze2.md [hunk 18]
```diff
@@ -0,0 +1,14 @@
...
+[down](../maze1/maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze2/maze2.md]: This link is also on the line directly after another link.

File: english/plugh/maze3/maze3.md [hunk 19]
```diff
@@ -0,0 +1,14 @@
+You are in a maze of little twisty passages, all different.
+
+Which way should we go?
+
+[north](../maze1/maze1.md)
+[south](../maze1/maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze3/maze3.md]: This link is also on the line directly after another link.

File: english/plugh/maze3/maze3.md [hunk 20]
```diff
@@ -0,0 +1,14 @@
...
+[west](../maze1/maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze3/maze3.md]: This link is also on the line directly after another link.

File: english/plugh/maze3/maze3.md [hunk 21]
```diff
@@ -0,0 +1,14 @@
...
+[east](../maze1/maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze3/maze3.md]: This link is also on the line directly after another link.

File: english/plugh/maze3/maze3.md [hunk 22]
```diff
@@ -0,0 +1,14 @@
...
+[northwest](../maze1/maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze3/maze3.md]: This link is also on the line directly after another link.

File: english/plugh/maze3/maze3.md [hunk 23]
```diff
@@ -0,0 +1,14 @@
...
+[northeast](../maze1/maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze3/maze3.md]: This link is also on the line directly after another link.

File: english/plugh/maze3/maze3.md [hunk 24]
```diff
@@ -0,0 +1,14 @@
...
+[southwest](../maze1/maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze3/maze3.md]: This link is also on the line directly after another link.

File: english/plugh/maze3/maze3.md [hunk 25]
```diff
@@ -0,0 +1,14 @@
...
+[southeast](../maze1/maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze3/maze3.md]: This link is also on the line directly after another link.

File: english/plugh/maze3/maze3.md [hunk 26]
```diff
@@ -0,0 +1,14 @@
...
+[up](../maze1/maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze3/maze3.md]: This link is also on the line directly after another link.

File: english/plugh/maze3/maze3.md [hunk 27]
```diff
@@ -0,0 +1,14 @@
...
+[down](../maze4/maze4.md)
```
Comment (casey-collab) [inline english/plugh/maze3/maze3.md]: This link is also on the line directly after another link.

File: english/plugh/maze4/maze4.md [hunk 28]
```diff
@@ -0,0 +1,14 @@
+You are in a maze of little twisting passages, all different.
+
+Which way should we go?
+
+[north](../maze1/maze1.md)
+[south](../maze1/maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze4/maze4.md]: This link is also on the line directly after another link.

File: english/plugh/maze4/maze4.md [hunk 29]
```diff
@@ -0,0 +1,14 @@
...
+[west](../maze5/maze5.md)
```
Comment (casey-collab) [inline english/plugh/maze4/maze4.md]: This link is also on the line directly after another link.

File: english/plugh/maze4/maze4.md [hunk 30]
```diff
@@ -0,0 +1,14 @@
...
+[east](../maze1/maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze4/maze4.md]: This link is also on the line directly after another link.

File: english/plugh/maze4/maze4.md [hunk 31]
```diff
@@ -0,0 +1,14 @@
...
+[northwest](../maze1/maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze4/maze4.md]: This link is also on the line directly after another link.

File: english/plugh/maze4/maze4.md [hunk 32]
```diff
@@ -0,0 +1,14 @@
...
+[northeast](../maze1/maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze4/maze4.md]: This link is also on the line directly after another link.

File: english/plugh/maze4/maze4.md [hunk 33]
```diff
@@ -0,0 +1,14 @@
...
+[southwest](../maze1/maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze4/maze4.md]: This link is also on the line directly after another link.

File: english/plugh/maze4/maze4.md [hunk 34]
```diff
@@ -0,0 +1,14 @@
...
+[southeast](../maze1/maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze4/maze4.md]: This link is also on the line directly after another link.

File: english/plugh/maze4/maze4.md [hunk 35]
```diff
@@ -0,0 +1,14 @@
...
+[up](../maze1/maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze4/maze4.md]: This link is also on the line directly after another link.

File: english/plugh/maze4/maze4.md [hunk 36]
```diff
@@ -0,0 +1,14 @@
...
+[down](../maze1/maze1.md)
```
Comment (casey-collab) [inline english/plugh/maze4/maze4.md]: This link is also on the line directly after another link.

**Comment by jdhj** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Unfortunately, pull requests adding more than 100 lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to add 100 lines or fewer.

**Comment by jdhj** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :icecream:

**Pull request merged**